tests/ui/regex.stderr

```diff
@@ -79,93 +94,110 @@ error: regex syntax error: unrecognized escape sequence
 LL |     let raw_string_error = Regex::new(r#"[...//...]"#);
    |                                              ^^
 
+error: regex parse error:
+           /b/c
+             ^^
+       error: unrecognized escape sequence
+  --> $DIR/regex.rs:40:42
+   |
+LL |     let escaped_string_span = Regex::new("/b/c");
+   |                                          ^^^^^^^^
+   |
+   = help: consider using a raw string literal: `r".."`
```

Comment:
compiletest is doing its windows path normalisation thing, it looks like this normally:

```
error: regex parse error:
           \b\c
             ^^
       error: unrecognized escape sequence
 --> ../t.rs:6:42
  |
6 |     let escaped_string_span = Regex::new("\\b\\c");
  |                                          ^^^^^^^^
  |
  = help: consider using a raw string literal: `r".."`
```